app/1.0/docs/release-notes.md

```diff
@@ -11,6 +11,107 @@ title: Release notes
   }
 </style>
 
+## [Release 1.6.0](https://github.com/polymer/polymer/tree/v1.6.0) (2016-06-29) {#v-1-6-0}
+
+New features:
+
+-   Added support for native custom CSS properties.
+
+    To use, set `lazyRegister: true` and `useNativeCSSProperties: true` in Polymer
+    settings (e.g. `<script> Polymer = {lazyRegister: true,
+    useNativeCSSProperties: true};</script>` before importing `polymer.html`).
+
+    Enabling the feature only impacts browsers that support native custom CSS
+    properties (Chrome and Firefox; Safari 9+ supports custom properties, but is
+    buggy and thus opted out for now).  On these browsers, normal custom CSS
+    properties such as `--custom-property: value` will use native properties;
+    [custom CSS mixins](https://www.polymer-project.org/1.0/docs/devguide/styling#custom-css-mixins)
+    such as `--custom-mixin: { ... }` and their `@apply` points of usage will be
+    destructured into individual custom properties once at registration time such
+    that native properties are used for all custom properties.  This eliminates
+    all _instance time_ work related to the custom property shim, which should
+    provide substantial performance gains.
+
+    *Caveats:* When native custom properties are in use, Polymer's
+    `updateStyles(...)` API is supported with one exception: calling
+    `updateStyles` to change the value of custom CSS mixins is not supported.
+
+-   Integrated support for
+    [polymer-css-build](https://github.com/PolymerLabs/polymer-css-build).
+
+    The CSS build performs Shady DOM CSS selector shimming as well as
+    destructuring custom CSS mixins to individual properties, which can help
+    eliminate all runtime cost of Shady DOM style shimming, as well as all
+    runtime cost of custom property shimming (on browsers with native CSS
+    properties).  Achieving optimal results may require selectively serving
+    different built CSS depending on client capabilities.  See that repo for
+    details on how to run the build.
+
+-   Added support for defining CSS custom properties in `@media` queries
+
+    The following use of CSS custom properties inside `@media` rules is
+    now supported, allowing for dynamic change of custom property values
+    based on screen size, etc.:
+
+    ```html
+    <style>
+      @media (max-width: 400px) {
+        :host {
+          --gutter-width: 16px;
```

Comment:
@robdodson Ah good point.  We need to add another caveat there.  Users will need to call `Polymer.updateStyles()` to re-evaluate the media queries upon window resize.

We didn't include a window resize listener in the core to allow the user to choose an appropriate strategy for calling `updateStyles`, since there is a performance/responsiveness tradeoff depending on whether it is called on every resize or throttled somehow.

Here's a jsBin that shows the basic code to add to an app to ensure styles are updated on window resize, throttled to one update per 100ms:
http://jsbin.com/bicazocaji/edit?html,output